src/App.tsx

```diff
@@ -1,24 +1,36 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { TodoFooter } from './components/todofooter/todofooter';
+import { TodoErr } from './components/todoerr/todoerr';
+import { TodoForm } from './components/todoform/todoform';
+import { TodoList } from './components/todolist/todolist';
+import { useTodos } from './context/todoProvider';
+import { USER_ID } from './utils/userID';
 
 export const App: React.FC = () => {
+  const { todos } = useTodos();
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">React Todo App - Load Todos</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <TodoForm />
+
+        <TodoList />
+
+        {todos.length > 0
```

Comment:
Check my comment in footer component. You won't need conditional rendering here ;)